.ci/windows-steps.yaml

```diff
@@ -66,6 +66,7 @@ steps:
     -DARMADILLO_LIBRARY="..\armadillo-8.400.0\Release\armadillo.lib" `
     -DBOOST_INCLUDEDIR=$(Agent.ToolsDirectory)\boost.1.60.0.0\lib\native\include `
     -DBOOST_LIBRARYDIR=$(Agent.ToolsDirectory)\boost_libs `
+    -DBUILD_JULIA_BINDINGS=OFF `
```

Comment:
Hm, that is strange looks like we are still trying to run the julia bindings tests.

Reply:
Yeah, the debugging exposed a bug in the CMake configuration. :smile: I think 3f07bcf fixes it.

Reply:
Nice, I guess with https://github.com/mlpack/mlpack/pull/2278 we should see more green builds.

Reply:
Here's hoping!  It's been a really bad week for our build systems and I think a good part of that is my fault. :smile: